Composer/packages/client/src/components/Header.tsx

```diff
@@ -95,33 +121,110 @@ export const Header = () => {
   const locale = useRecoilValue(localeState(projectId));
   const appUpdate = useRecoilValue(appUpdateState);
   const { showing, status } = appUpdate;
+  const [showStartBotsPanel, setStartPanelView] = useState(false);
+  const runningBots = useRecoilValue(runningBotsSelector);
+  const [startPanelText, setStartPanelText] = useState('');
+  const { projectIds: projectIdsRunning } = runningBots;
+  const [areBotsStarted, setBotsInBotProjectStarted] = useState<boolean>(false);
+  const { stopAllBots, startAllBots } = useLocalBotOperations();
+  const [showStartBotsWidget, setStartBotsWidgetVisible] = useState(true);
+  const currentMode = useRecoilValue(currentModeState);
```

Comment:
I think we also have a`useLocation` hook

Reply:
Yep. Initially I was using it but just decided to use our local state